File: app/src/main/java/me/sweetll/evilhide/SettingsActivity.java
package me.sweetll.evilhide;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.FrameLayout;

import butterknife.Bind;
import butterknife.ButterKnife;
import me.sweetll.evilhide.fragment.SettingFragment;

/**
 * Created by sweet on 3/7/16.
 */
public class SettingsActivity extends AppCompatActivity {

    @Bind(R.id.content) FrameLayout content;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);

        ButterKnife.bind(this);

        getFragmentManager().beginTransaction()
                .replace(android.R.id.content, new SettingFragment())
                .commit();
    }
}

File: app/src/main/kotlin/me/sweetll/evilhide/SettingsActivity.kt
package me.sweetll.evilhide

import android.databinding.DataBindingUtil
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import me.sweetll.evilhide.databinding.ActivitySettingsBinding
import me.sweetll.evilhide.fragment.SettingFragment

class SettingsActivity : AppCompatActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding : ActivitySettingsBinding = DataBindingUtil.setContentView(this, R.layout.activity_settings)
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, SettingFragment())
                .commit()
    }
}
